File: app/src/main/java/moe/tlaster/openween/core/model/types/RepostType.java
package moe.tlaster.openween.core.model.types;

/**
 * Created by Tlaster on 2016/9/2.
 */
public enum RepostType {
    None(0),
    CommentCurrentWeibo(1),
    CommentOriginWeibo(2),
    CommentAll(3),;

    private final int value;
    public int getValue() {
        return value;
    }
    RepostType(int value) {
        this.value = value;
    }
}

File: app/src/main/kotlin/moe/tlaster/openween/core/model/types/RepostType.kt
package moe.tlaster.openween.core.model.types

/**
 * Created by Tlaster on 2016/9/2.
 */
enum class RepostType constructor(val value: Int) {
    None(0),
    CommentCurrentWeibo(1),
    CommentOriginWeibo(2),
    CommentAll(3)
}
